content/applications/hr/fleet/configuration.rst

```diff
@@ -0,0 +1,254 @@
+   contracts appear in the list. Click on a :guilabel:`Contract` to view it.
+
+   An individual contract can also be found by navigating to :menuselection:`Fleet app --> Fleet -->
+   Fleet` and clicking on an individual vehicle. On the vehicle form, click the
+   :guilabel:`Contracts` smart button at the top of the page. The contract(s) associated with this
+   vehicle only appears in the list. Click on an individual contract to open it. The
+   :guilabel:`Responsible` person is listed on the contract.
+
+New Vehicle Request
+-------------------
+
+Set a limit on how many new vehicles are requested by setting a limit on requests based on fleet
+availability. When an employee is filling out the salary configurator form (after being offered a
+position), employees will *not* be able to request a new car if there are more than a specific
+number of cars available (vehicles that do not have a current driver assigned to them). Enter that
+specific number limit for existing available cars in this field.
+
+.. image:: configuration/fleet-settings.png
+   :align: center
+   :alt: Settings available for the Fleet application.
+
+.. example::
+   If the :guilabel:`New Vehicle Request` limit is set to 20 vehicles, and there are 25 vehicles
+   available, an employee would not be able to request a new vehicle. If there are only 10 cars
+   available, then the employee would be able to request a new vehicle.
+
+Manufacturers
+=============
+
+Odoo Fleet comes pre-configured with sixty-six commonly used car manufacturers in the database,
+along with their logos. To view the pre-loaded manufacturers, go to :menuselection:`Fleet app -->
+Configuration --> Manufacturers`.
+
+The list of manufacturers appear in alphabetical order. Each manufacturer's card lists how many
+specific models are configured for each particular manufacturer. Odoo comes with forty five
+pre-configured vehicle models from four major auto manufacturers: Audi, BMW, Mercedes, and Opel.
+
+.. image:: configuration/manufacturer.png
+   :align: center
+   :alt: Manufacturer card with the amount of models listed.
+
+Add a manufacturer
+------------------
+
+To add a new manufacturer to the database, click :guilabel:`Create`. A manufacturer form will
+load. Only two pieces of information are needed, the :guilabel:`Name` of the manufacturer, and the
+logo. Type the name of the manufacturer in the name field, and select an image to upload for the
+logo. When the information is entered, click :guilabel:`Save`.
+
+Vehicle Models
+==============
+
+Many manufacturers have a variety of models that they produce. When adding a vehicle to the fleet,
+it is important to specify the vehicle model being added. Odoo comes with pre-configured models from
+four manufacturers: Audi, BMW, Mercedes, and Opel. If a vehicle *other* than the pre-configured
+models from these manufacturers are part of a fleet, the model (and/or manufacturer) will need to be
+added to the database.
```

Comment:
Added Eddy Merckx
```suggestion
it is important to specify the vehicle model being added. Odoo comes with pre-configured models five
major auto manufacturers: Audi, BMW, Eddy Merckx, Mercedes, and Opel. If a vehicle *other* than the
pre-configured models from these manufacturers are part of a fleet, the model (and/or manufacturer)
will need to be added to the database.
```

Reply:
Eddy Merckx is actually a bicycle manufacturer, and the pre-loaded model is a bike, not a car. I am updating this to show that.